### ISSUE TITLE
Please consider changing how you run the github side of things

### ISSUE THREAD

**Issue by robclancy** [ckeditor/ckeditor5#6014]
Initially you have had a repository per plugin and then had issues on each one. You then redirected people to different areas to put the issue in the right spot (I've seen things moved etc). Then you realized it's a terrible idea to do this because github doesn't work well with issues in general, so best to keep them all in one spot and they got moved back. But now you have a separation of issues and pull requests / code. 
Also working on ckeditor5 to try make a custom fork or plugins is convoluted using your tools to make a directory with all repos etc... with basically what should be in this repository.

Everything looks backwards. Instead of having tools to setup a central working directory just make this repository to contain all the code and split it out with scripts.

You also match versions now and it is forced because if wrong versions you get the module error. There is no benefit of using multiple plugin repositories anymore.

For example Laravel is all in one directory, then split when needed on a cron job. https://github.com/laravel/framework/tree/6.x/bin

lodash does this in some way.

Looks like symfony does it https://github.com/symfony/symfony/tree/master/.github

Basically what would make things far easier to understand and work on, from the open source perspective, is to just be able to do it right here. Have a `packages` folder which contains every plugin. Those plugins get split out into their own repositories nightly and on release. The builds would stay in their own repositories as they are now, but with issues enabled on their repositories.

**Comment by Reinmar** [comment]
Thanks for the ideas. We are fully aware that the current setup isn't the simplest thing to understand and work with. It's been a challenge for us too. But it has its pros as well and ideologically works for us best. There are also other projects which work the same way, so we're not completely alone here :D

We're not open, though, to new ideas and improving the environment. The times change too. We'll definitely keep your feedback in mind.
